infra/base-images/base-builder-new/install_swift.sh

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+#!/bin/bash
+
+# generic swift
+apt-get update && apt install -y wget \
+           binutils \
+           libc6-dev \
+           libcurl3 \
+           libedit2 \
+           libgcc-5-dev \
+           libpython2.7 \
+           libsqlite3-0 \
+           libstdc++-5-dev \
+           libxml2 \
+           pkg-config \
+           tzdata \
+           zlib1g-dev
+
+wget https://swift.org/builds/swift-5.3.3-release/ubuntu1604/swift-5.3.3-RELEASE/swift-5.3.3-RELEASE-ubuntu16.04.tar.gz
+tar xzf swift-5.3.3-RELEASE-ubuntu16.04.tar.gz
+cp -r swift-5.3.3-RELEASE-ubuntu16.04/usr/* /usr/
+
+# generic swift symbolizer
+apt-get update && apt-get install -y build-essential make cmake ninja-build git python3 g++-multilib binutils-dev zlib1g-dev --no-install-recommends
+
+git clone --depth 1 https://github.com/llvm/llvm-project.git
```

Comment:
nit: The script should probably delete the source checkout as well as dependencies used to build llvm-symbolizer at the end (we do this in base-clang, but it's more complex here because I think some of these packages are meant to be left over for inheritors of base-builder-swift).

Reply:
Yeah, I played around a little with removing some -- some of them for sure couldn't but I'll keep trying!